### DISCUSSION TITLE
Kvantum release on Papyrus Ubuntu PPA does not get new versions

### DISCUSSION THREAD

**Discussion by ashleysommer** [tsujan/Kvantum#988] [General]
The Papirus Team PPA still has an old version of Kvantum (v1.0.10) it has not been updated with recent versions v1.1.0, v.1.1.1, v1.1.2).

How do we get in contact with that team to politely ask them to include a newer release?

I've opened an issue over at the papirus Development Team team Github too:
https://github.com/PapirusDevelopmentTeam/arc-kde/issues/146

**Comment by tsujan** [comment]
> The Papirus Team PPA still has an old version of Kvantum

I'm afraid they can't hear you here. This is the place where Kvantum is developed, independently of all distros and against the latest stable version of Qt.

Kvantum 1.1.0-1.1.2 is mostly about backward-incompatible changes in Qt ≥ 6.6. Maybe those versions of Qt aren't present in Ubuntu yet?

> I've opened an issue over at the papirus Development Team team Github

Yes, that may be a better place.

**Reply by ashleysommer** [reply]
>  changes in Qt ≥ 6.6 Maybe those versions of Qt aren't present in Ubuntu yet?

You're right, they are not in a mainline release of Ubuntu, however I am using the KDE Neon distro, their latest stable release is based on Ubuntu 22.04, but has Plasma 6.1 and Qt v6.7, that is why I'm looking to install the new version of Kvantum.
